draft-ietf-v6ops-464xlaton.xml

```diff
@@ -407,11 +423,11 @@ OLD TEXT:
    The NAT64 prefix could be discovered by means of the method defined
    in <xref target="RFC7050"/> only if the service provider uses DNS64 [RFC6147].  It
```

Comment:
Does the [RFC6147] need to be replaced here too? I see you replaced it with some XML below. I'm a simple "I write drafts in markdown" soul who doesn't always get what is needed when handcrafting XML.

Reply:
Yes, it does, will fix it.